src/components/Article.jsx

```diff
@@ -17,27 +17,17 @@ function Article() {
     });
   }, []);
 
-  const handleClickVoteUp = () => {
-    patchArticle(article_id, 1)
-      .then((updatedArticle) => {
-        setArticleVotes((currentArticleVotes) => {
-          return currentArticleVotes + 1;
-        });
-      })
-      .catch((err) => {
-        setApiError(err.response);
-      });
-  };
+  const handleClick = (event) => {
+    setArticleVotes((currentArticleVotes) => {
+      return currentArticleVotes + event;
+    });
 
-  const handleClickVoteDown = () => {
-    patchArticle(article_id, -1)
-      .then((updatedArticle) => {
-        setArticleVotes((currentArticleVotes) => {
-          return currentArticleVotes - 1;
-        });
+    patchArticle(article_id, event)
+      .then(() => {
+        setApiError(null);
       })
       .catch((err) => {
-        setApiError(err.response);
+        setApiError(err);
       });
```

Comment:
If a request fails your current implementation breaks. I would suggest taking a look at the err object you get back to see what this actually looks like.